conf/bridge-server.conf

```diff
@@ -1,4 +1,4 @@
-channel.throttle.max.requests = 2
+channel.throttle.max.requests = 1
```

Comment:
We might see bugs about this. It's mysterious that when I ask for it, I don't always get a message and it's not clear then if the old message is still valid or not.

Reply:
Email complaints suggest this is happening already. Someone will request sign-in, and then they will click the link several hours later when it already expires, which generates a new sign-in link, and then they mark both emails as spam.

Given that our complaint rate is dangerously high, I think we should err on the side of sending too few emails rather than too many emails.

If emails are in fact not being delivered, we have the logs to verify this.